lib/compat/wordpress-6.5/block-bindings/class-wp-block-bindings-registry.php

```diff
@@ -0,0 +1,197 @@
+<?php
+/**
+ * Block Bindings API: WP_Block_Bindings_Registry class.
+ *
+ * Support for overriding content in blocks by connecting them to different sources.
+ *
+ * @package WordPress
+ * @subpackage Block Bindings
+ * @since 6.5.0
+ */
+
+if ( class_exists( 'WP_Block_Bindings_Registry' ) ) {
```

Comment:
Maybe try moving this check to the place where the class gets included - `load.php`.